src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -83,44 +84,59 @@ private void createContactInChips(Appeal appeal, String userId) {
     }
 
     protected ChipsContact buildChipsContact(Appeal appeal) {
```

Comment:
This should be private

Reply:
This will be fixed when the contact description code is extracted into a utility class in the next PR.